io/disk/softwareraid.py

```diff
@@ -86,6 +81,11 @@ def extensivetest(self):
         Extensive software raid options are run viz create, delete, assemble,
         create spares, remove and add drives
         """
+        self.disk = self.disk.split(" ")
+        self.sparedisk = self.disk.pop()
+        self.remadd = ''.join(self.disk[-1:])
+        self.disk_count = len(self.disk)
+        self.disk = ' '.join(self.disk)
```

Comment:
For basic test, we do not need spare disk. So, direct disk count is ok.
For advanced levels, we need spare disk. So, after removing spare disk from the main list of disks, we are calculating disk count.